OWNERS

```diff
@@ -0,0 +1,14 @@
+# This OWNERS file is not interpreted by any tooling in MetalLB,
```

Comment:
Don't we want to name this CODEOWNERS and use this github feature? https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

Maybe what github does is enough for us and we don't need the bots that k8s has? What do you think?

I haven't used CODEOWNERS much with github, so I don't really know if it will fall short easily. But I'd prefer to use something built-in, as it seems simpler, if it does what we need :)

Reply:
Ah, I'll take a look at that.  I hadn't seen it before, so just went with a kube convention.

Reply:
Awesome, thanks! :)

Reply:
@rata It looks like `CODEOWNERS` can reference a github team.  If we can create a `metallb-maintainers` team, then it can be referenced in this file, but also mentioned in PRs or issues to notify the whole team if appropriate.  I need someone who has admin access to the metallb org to try out creating a team and making it visible so we can see if this works, though.  Maybe ping me on slack sometime when you want to look at it with me?

Reply:
It can, but also individuals (just having a look: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax). If we want to be explicit about usernames, maybe have the maintainers individuals gh handles owners for all the code?

Reply:
Sure, I'll put individuals names in there as a start.  I thought teams might be a bit more discoverable, but that could easily be done later if we feel like it.  Example: https://github.com/orgs/kubernetes/teams

Reply:
... though now I remember I'm a member of the kube github org.  The teams list may not be visible if you're not an org member.  That's what the docs imply, anyway.

I'll push a change with indiividuals in a bit

Reply:
done!